wasm/README.md

```diff
@@ -18,10 +18,7 @@ gunzip models/*/*
 ```js
 // The model configuration as YAML formatted string. For available configuration options, please check: https://marian-nmt.github.io/docs/cmd/marian-decoder/
 // This example captures some of the most relevant options
-const modelConfig = `vocabs:
-  - /esen/vocab.esen.spm
-  - /esen/vocab.esen.spm
-beam-size: 1
+const modelConfig = `beam-size: 1
```

Comment:
Can't this be an `example.js` loaded and linked here? Why do you have two copies.. I remember reviewing the same yesterday.

Reply:
The goal here is to provide an idea on how to use the JS APIs. The code that you had reviewed yesterday is a fully functional wasm test page :)